languages/polish/pack/src/main/res/xml/polish_azerty.xml

```diff
@@ -16,7 +16,7 @@
 
     <Row>
         <Key android:codes="q" android:keyEdgeFlags="left" android:popupCharacters="" />
-        <Key android:codes="s" android:popupCharacters="" />
+        <Key android:codes="s" android:popupKeyboard="@xml/polish_popup_s" />
```

Comment:
If there is only one character, and no special layout needs, you can use the `android:popupCharacters` attribute
```
android:popupCharacters="ś"
```

Reply:
I was reproducing the way @martholomew did it, maybe he can explain his reasoning?

Reply:
@MegaArthur The reason I do it that way is because I'm [generating the keyboard files](https://github.com/martholomew/ezlanguagepack) and I didn't feel like changing how the xml file is made if there's more than one or some-odd popup keys. I also encountered some bugs with popupCharacters.

Reply:
It also consolidates the popups into one file to make it easier to change for all of the keyboards, I think that it's fine and works out good.

Reply:
I guess that settles it, @menny?